tests/sentry/notifications/utils/test_participants.py

```diff
@@ -672,9 +672,70 @@ def test_fallthrough_all_members_multiple_teams(self):
         }
 
     @with_feature("organizations:issue-alert-fallback-targeting")
-    def test_fallthrough_admin_or_recent(self):
+    def test_fallthrough_admin_or_recent_under_20(self):
         event = self.store_event("admin.lol", self.project)
-        assert (
-            self.get_send_to_fallthrough(event, self.project, FallthroughChoiceType.ADMIN_OR_RECENT)
-            == {}
-        )
+        assert self.get_send_to_fallthrough(
+            event, self.project, FallthroughChoiceType.ADMIN_OR_RECENT
+        ) == {
+            ExternalProviders.EMAIL: {
+                UserService.serialize_user(self.user),
+                UserService.serialize_user(self.user2),
+            }
+        }
+
+    @with_feature("organizations:issue-alert-fallback-targeting")
+    def test_fallthrough_admin_or_recent_admin_only(self):
+        notified_users = []
+        for i in range(21):
+            new_user = self.create_user(email=f"user_{i}@example.com", is_active=True)
```

Comment:
I don't think you need to specify if the admin is_active since the code only considers it for non-admin members.